de.metas.business/src/main/java/org/adempiere/impexp/bpartner/BPCreditLimitImportHelper.java

```diff
@@ -0,0 +1,78 @@
+package org.adempiere.impexp.bpartner;
+
+import java.math.BigDecimal;
+
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.util.time.SystemTime;
+import org.compiere.model.I_C_BPartner;
+import org.compiere.model.I_C_BPartner_CreditLimit;
+import org.compiere.model.I_I_BPartner;
+
+import lombok.NonNull;
+
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2017 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+/* package */ class BPCreditLimitImportHelper
+{
+	private final int Management_C_CreditLimit_Type_ID = 540001;
+	private final int Insurance_C_CreditLimit_Type_ID = 540000;
+
+	public static BPCreditLimitImportHelper newInstance()
+	{
+		return new BPCreditLimitImportHelper();
+	}
+
+	private BPCreditLimitImportHelper()
+	{
+	}
+
+	public final void importRecord(final I_I_BPartner importRecord)
+	{
+		I_C_BPartner_CreditLimit bpCreditLimit = null;
```

Comment:
don't we need to handle that case that there is already a credit limit that just needs to be updated?

Reply:
Yes, you are right!